### PR TITLE
History

### PR DESCRIPTION
**Pull request by YehyaTamimi**
this branch took a lot more than it should have due to some issue in the git pull and git rebase commands, that made me rewrite some of the code again.

### CODE REVIEW

**Comment by YehyaTamimi** [comment]
> image keep the entered value in the input

Implemented in the main branch.

**Comment by DuaaH** [comment]
> > image keep the entered value in the input
> 
> Implemented in the main branch.

good, so solve the conflict

File: Js/search.js [hunk 1]
```diff
@@ -111,18 +227,87 @@ const createRoomsElement = () => {
...
+    roomTypes.forEach((type) => {
+        let rooms = document.querySelector(type);
+        let roombuttons = rooms.querySelectorAll("button");
+        roombuttons.forEach((button) => {
+            button.addEventListener('click', () => {
+                roombuttons.forEach(function (btn) {
+                    btn.classList.remove('active');
+                });
+                button.classList.add('active');
+            });
```
Comment (DuaaH) [inline Js/search.js:249]: use arrow function

**Comment by YehyaTamimi** [comment]
> > > image keep the entered value in the input
> > 
> > 
> > Implemented in the main branch.
> 
> good, so solve the conflict

any other changes required before solving the conflicts?

**Comment by DuaaH** [comment]
> > > > image keep the entered value in the input
> > > 
> > > 
> > > Implemented in the main branch.
> > 
> > 
> > good, so solve the conflict
> 
> any other changes required before solving the conflicts?

nope, I want to check it with the solving conflict

**Comment by YehyaTamimi** [comment]
> > > > > image keep the entered value in the input
> > > > 
> > > > 
> > > > Implemented in the main branch.
> > > 
> > > 
> > > good, so solve the conflict
> > 
> > 
> > any other changes required before solving the conflicts?
> 
> nope, I want to check it with the solving conflict

you can check now

**Pull request merged**